pkg/plugin/connector/standalone/registry.go

```diff
@@ -60,30 +64,32 @@ func NewRegistry(logger log.CtxLogger, pluginDir string) *Registry {
 			r.logger.Warn(context.Background()).Err(err).Msg("could not extract absolute connector plugins path")
 		} else {
 			r.pluginDir = absPluginDir // store plugin dir for hot reloads
-			r.reloadPlugins()
 		}
 	}
 
-	r.logger.Info(context.Background()).
-		Str(log.PluginPathField, r.pluginDir).
-		Int("count", len(r.List())).
-		Msg("standalone connector plugins initialized")
-
 	return r
 }
 
-func (r *Registry) reloadPlugins() {
```

Comment:
Reload plugins was created purposefully, so we could create a goroutine that calls it repeatedly with a sleep interval, this way we could hot-reload plugins that are dropped in the folder while Conduit is running. Not a big thing though, we can easily extract it when we add that, just explaining why it was there.